fractal-puppy/Engine
# Improved enemies scripts

Improved the enemies behaviour:
- Now every enemy uses the nav mesh to move.
- Cleaned unnecessary code and added some comments.
- Fixed some bugs.

To test:
- Make sure to recompile all the scripts before testing.
- Load "EnemiesTest" scene.
- Generate the nav mesh if not already.
- Press play and check that the enemy behaviour works.
- Check that the enemies avoid the holes of the nav mesh.
